README.md

```diff
@@ -47,4 +47,25 @@ pipelines, err := client.GetPipelines(app.Name)
 // etc...
 ```
 
+## Development
 
+Build the project:
+
+```
+$ go build
+```
+
+Test the project:
+
+```
+$ go test
+```
+
+### Cutting a New Release
+
+1. Update the [CHANGELOG.md]'s `##Unreleased` section with the next version and
+today's date in `YYYY-MM-DD` format.
+  - Don't forget to update the release URL so that it's easy to diff what was
+  changed (look at the bottom of the CHANGELOG for existing examples).
+1. `git tag vx.x.x` where `x.x.x` is the version you're releasing,
```

Comment:
I didn't know that just creating tags automatically creates github releases!

Reply:
It's basically just a tarball of the repo at the time you add the tag, if you want to customize the release artifacts at all you still have to use something like GitHub Actions haha. For `plank`, it's less important because there's no binary or anything 🤷‍♂